pkg/infrastructure/mgmt_console.go

```diff
@@ -24,8 +24,8 @@ const (
 	DefaultMgmtConsoleName = "management-console"
 	// DefaultMgmtConsoleImageName ...
 	DefaultMgmtConsoleImageName = "kogito-management-console"
-	// DefaultMgmtConsoleImageFullTag ...
-	DefaultMgmtConsoleImageFullTag = "quay.io/kiegroup/" + DefaultMgmtConsoleImageName + ":latest"
+	// DefaultMgmtConsoleImageNoVersion is the default Management Console image name without version definition
+	DefaultMgmtConsoleImageNoVersion = "quay.io/kiegroup/" + DefaultMgmtConsoleImageName + ":"
```

Comment:
I think it makes no sense to have this field like this any longer since we cannot use it by its own (we require the tag version always).
I would suggest to have a method instead:

```go
GetMgmtConsoleImageVersion(tag string) {
    return "quay.io/kiegroup/" + DefaultMgmtConsoleImageName + ":" + tag;
}
```

This is only a suggestion! The same for the rest of services.

Reply:
I'd say that `quay.io/kiegroup/` could be in an internal constant in the `infrasctructure` package as well. Let's keep this way for now and refactor in the feature since we're targeting 0.9.x and I've done too much changes already :sweat_smile:

Reply:
Okis!